src/ratbag_emu/protocol/base.py

```diff
@@ -29,10 +28,10 @@ def __init__(self, owner, rdesc=None, info=None, name='Generic Device',
 
         self.owner = owner
 
-        self.info = info
+        self.info = owner.info
```

Comment:
might be easier to implement this as a property:
```python
@property
def info(self):
    return self.owner.info
```

Reply:
Sure.